vendor/bat-native-ads/src/bat/ads/internal/ad_serving/inline_content_ads/inline_content_ad_serving.cc

```diff
@@ -79,7 +108,54 @@ void AdServing::MaybeServeAd(const std::string& dimensions,
 
         eligible_ads_->SetLastServedAd(ad);
 
-        InlineContentAdInfo inline_content_ad = BuildInlineContentAd(ad);
+        inline_content_ad = BuildInlineContentAd(ad);
+
+        BLOG(1, "Serving inline content ad:\n"
+                    << "  uuid: " << inline_content_ad.uuid << "\n"
+                    << "  creativeInstanceId: "
+                    << inline_content_ad.creative_instance_id << "\n"
+                    << "  creativeSetId: " << inline_content_ad.creative_set_id
+                    << "\n"
+                    << "  campaignId: " << inline_content_ad.campaign_id << "\n"
+                    << "  advertiserId: " << inline_content_ad.advertiser_id
+                    << "\n"
+                    << "  segment: " << inline_content_ad.segment << "\n"
+                    << "  title: " << inline_content_ad.title << "\n"
+                    << "  description: " << inline_content_ad.description
+                    << "\n"
+                    << "  imageUrl: " << inline_content_ad.image_url << "\n"
+                    << "  dimensions: " << inline_content_ad.dimensions << "\n"
+                    << "  ctaText: " << inline_content_ad.cta_text << "\n"
+                    << "  targetUrl: " << inline_content_ad.target_url);
+
+        NotifyDidServeInlineContentAd(inline_content_ad);
+
+        callback(/* success */ true, dimensions, inline_content_ad);
+      });
+}
+
+void AdServing::MaybeServeAdV2(const std::string& dimensions,
+                               GetInlineContentAdCallback callback) {
+  InlineContentAdInfo inline_content_ad;
```

Comment:
Seems like some changes wasn't applied, because of unused variable.

Reply:
fixed